### PR TITLE
doc(github): add issue templates

### PR DESCRIPTION
**Pull request by henyxia**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/organization-question.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+name: Organization question
+about: Want to change something about how the project works?
+title: ''
+labels: question
+assignees: ''
+
+---
+
+Feel free to describe what you want on the organization :)
```
[the remark this answers is not in the document]
Comment (henyxia) [reply .github/ISSUE_TEMPLATE/organization-question.md]: Fixed

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
```
[the remark this answers is not in the document]
Comment (henyxia) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Fixed

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+name: Feature request
+about: Suggest an idea for this project
+title: "[FEAT]"
+labels: feature
+assignees: ''
+
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```
[the remark this answers is not in the document]
Comment (henyxia) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Fixed

**Comment by henyxia** [comment]
The `assignees: ''` blocks only means that they are not a default assignee :)

**Comment by Fraise** [comment]
Looks good to me!

**Pull request merged**